test/AutoRemediateS3-002.test.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
+
+const source = require('../functions/AutoRemediateS3-002');
+
+//const config = require('./config');
+const AWS = require("aws-sdk");
+
+const CCRuleCode = 'S3-002';
+const CCRuleName = 'BucketPublicReadAcpAccess';
+const allUsersURI = 'http://acs.amazonaws.com/groups/global/AllUsers';
+const readAcpPermission = "READ_ACP";
+const aclSkeleton = '{"Owner":"", "Grants":[]}'; // skeleton for new permission grants
+
+describe('S3-002 READ_ACP auto-remediation', () => {
+
+    describe('S3 ACL Grants shall transfer to new ACL except for READ_ACP grant', () => {
+
+        describe('S3 ACL with READ_ACP grant', () => {
+            it('is not transfered to new ACL', () => {
+                var aclNew = JSON.parse(aclSkeleton);
+                const readAcpGrant = JSON.parse('{ "Grantee": { "Type": "Group", "URI": "http://acs.amazonaws.com/groups/global/SomeUsers" }, "Permission": "READ_ACP" }');
```

Comment:
`http://acs.amazonaws.com/groups/global/SomeUsers` isn't a valid predefined user group, as per: https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html

Is this intentional?

I would expect that in this scenario it would not be filtered out